src/Components/ArticlePage.js

```diff
@@ -0,0 +1,98 @@
+import "./ArticlePage.css";
+import { useParams } from "react-router-dom";
+import {
+  getArticleById,
+  getArticleCommentsById,
+  patchArticleVote,
+} from "../Api/Api";
+import React, { useEffect, useState } from "react";
+import Comment from "./Comment";
+import thumbsDown from "../Assets/thumbs-down-outline.svg";
+import thumbsUp from "../Assets/thumbs-up-outline.svg";
+
+const ArticlePage = () => {
+  const { id } = useParams();
+
+  const [articleComments, setArticleComments] = useState([]);
+
+  const [article, setArticle] = useState(null);
+
+  // fetches articles if none have been passed through state or props using the id from params to obtain correct article
+
+  useEffect(
+    () => {
+      if (!article) {
+        const fetchArticles = async () => {
```

Comment:
Is this needed? If you have clicked on an article from the list then the articles are already there? So this IF statement is redundant. You only need to make a call to the api by article id.